src/web/extractors.rs

```diff
@@ -66,6 +66,15 @@ pub struct UidParam {
     uid: u64,
 }
 
+pub fn clean_element(v: &str) -> Result<String, urlencoding::FromUrlEncodingError> {
+    // Some systems are wrapping elements with "{}". While not strictly prohibited,
+    // and there are no cases in the wild , it  may cause some problems for data
+    // storage and retrival. Normalize these by removing these charcters.
+    let decoded = urlencoding::decode(v)?;
+    let fixed = decoded.replace("\"", "").replace("{", "").replace("}", "");
```

Comment:
Why the replace? I don't think we want to mess with the actual value that was specified -- that's the original problem here.

E.g. though `PUT` gets `bso_id` from the path (this would convert e.g. `{password-id}` to `password-id` , a `POST` doesn't, getting it from the payload: updating ```'id': '{password-id}'``` would touch a different `bso_id` of `{password-id}`

Reply:
Ah, I thought the goal was to do that, in which case I'd need to make sure POST does the same thing. If we're just going to flat out reject it, then yeah, returning an error makes more sense.